src/main/scala/org/apache/mesos/chronos/scheduler/api/JobManagementResource.scala

```diff
@@ -84,6 +84,36 @@ class JobManagementResource @Inject()(val jobScheduler: JobScheduler,
                   case _ =>
                 }
             }
+          case t: TriggeredJob =>
```

Comment:
The workaround here would be to be schedule a job far in the future. I'd rather not add an additional type here unless necessary. Would this workaround be acceptable?